src/Auth0.AspNetCore.Authentication/Auth0AuthenticationApiClient/Auth0AuthenticationApiClient.cs

```diff
@@ -0,0 +1,152 @@
+using System;
+using System.Collections.Generic;
+using System.Threading;
+using System.Threading.Tasks;
+
+using Auth0.AuthenticationApi;
+using Auth0.AuthenticationApi.Models;
+using Auth0.AuthenticationApi.Models.Ciba;
+using Auth0.AuthenticationApi.Models.Mfa;
+using Models = Auth0.AuthenticationApi.Models;
+
+namespace Auth0.AspNetCore.Authentication.Auth0AuthenticationApiClient;
+
+/// <inheritdoc cref="IAuth0AuthenticationApiClient"/>
+public class Auth0AuthenticationApiClient : IAuth0AuthenticationApiClient
```

Comment:
We have added the Auth0 prefix to not collide with the underlying `IAuthenticationApiClient`, but that seems awkward as that makes me wonder, is `IAuthenticationApiClient` itself not Auth0 specific?